modules/ROOT/pages/managing-via-the-ops-center.adoc

```diff
@@ -87,7 +87,7 @@ image::ops-center-settings.png[]
 |Key | Description
 |`name` | Specifies the name of the log forwarding rule.
 |`address` | Specifies the endpoint and port to forward the log data
-|`protocol` | Specifies the protocol to send the data to. Supported protocols are TCP or UDP.
+|`protocol` | Specifies the protocol to send the data to. Supported protocols are TCP or UDP. Defaults to `tcp`.
```

Comment:
Is this a list that has the values preset? If it is a field, it is not clear what the user enters for UDP. Is it udp?